library/std/src/panic.rs

```diff
@@ -316,16 +316,18 @@ where
 ///
 /// # Notes
 ///
-/// Note that this function **might not catch all panics** in Rust. A panic in
-/// Rust is not always implemented via unwinding, but can be implemented by
-/// aborting the process as well. This function *only* catches unwinding panics,
-/// not those that abort the process.
-///
-/// Note that if a custom panic hook has been set, it will be invoked before
-/// the panic is caught, before unwinding.
-///
-/// Also note that unwinding into Rust code with a foreign exception (e.g.
-/// an exception thrown from C++ code) is undefined behavior.
+/// This function **might not catch all panics** in Rust. A panic in Rust is not
+/// always implemented via unwinding, but can be implemented by aborting the
+/// process as well. This function *only* catches unwinding panics, not those
+/// that abort the process.
+///
+/// If a custom panic hook has been set, it will be invoked before the panic is
+/// caught, before unwinding.
+///
+/// Although unwinding into Rust code with a foreign exception (e.g. an
+/// exception thrown from C++ code) via an appropriate ABI (e.g. `"C-unwind"`)
+/// is permitted, catching such an exception using this function is undefined
+/// behavior.
```

Comment:
Removing the `Note that... Note that... Also note that...` verbiage is technically unrelated to `c_unwind`, but it seemed clumsy and redundant to me. Happy to revert that change if someone feels otherwise.

Reply:
+1 to this